pkg/storage/postgresql/postgresql.go

```diff
@@ -15,8 +15,8 @@ import (
 )
 
 type (
-	// DB holds the database connection.
-	DB struct {
+	// PostgresStore holds the database connection.
+	PostgresStore struct {
```

Comment:
Same comment as the one the related to the `AWSSecrets` I would not use the construction: 
```go
type (
)
```

Reply:
Fixed in commit 3af143f2bb182c8b1fc75a9674b2ca45f7a95726